docs/src/man/getting_started.md

```diff
@@ -26,6 +26,20 @@ Throughout the rest of this tutorial, we will assume that you have installed the
     See [here](https://jupyter-client.readthedocs.io/en/stable/kernels.html) for information about location
     and specification of Jupyter kernels.
 
+
+## DataFrame Introduction
+ 
+A DataFrame is a table that can contain data of various types side-by-side. It is a two-dimensional data structure with three main components: rows, columns, and the data:
+
+![DataFrame_Concept](https://user-images.githubusercontent.com/8378161/80563049-9ec88180-899e-11ea-87da-a850bd24bb5d.png)
+
+In contrast to a matrix which can only be referenced by row and column indices, the columns of DataFrames have _names_ or _labels_ attached to them. The columns in a DataFrame can be referenced by name (and also by integer indexes).  In the example above, the names of the columns are Title, Author, PublishDate, Inventory, and Price. 
+
+Each column can, and usually does, have a  different data type. In the example table above, you see the datatypes of strings, dates, integers, and real numbers. 
```

Comment:
If you update the screenshot to Jupyter Notebook output then you can reference to the types that are shown in the screenshot exactly, like `String`, `Int` etc.